src/components/Select/Select.mdx

```diff
@@ -31,17 +31,20 @@ import { changelogData } from "./selectChangelogData";
 <Description of={SelectStories} />
 
 The `Select` component renders a `select` element along with its `option`
-children. For optimal accessibility, the `labelText` property is a required
-prop, regardless of the label visibility. Additionally, while the `id` prop is
-optional, a unique `id` attribute is necessary for accessibility. If the prop
-is left blank, a value will be generated for you.
+children. For a `Select` to be submitted as form data, a `name` attribute must
+be passed.
+
+For optimal accessibility, the `labelText` property is a required prop, regardless of
+the label visibility. Additionally, while the `id` prop is optional, a unique `id`
+attribute is necessary for accessibility. If the prop is left blank, a value will be
```

Comment:
I don't think the last part is true -- we don't seem to generate an `id` if it's blank. I don't think this needs to be addressed as part of this PR but it stood out while reviewing.

Reply:
You're right. I didn't add this either. Should I just remove that last sentence for now?